.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = hw-latest/caliptra-rtl
 	url = https://github.com/chipsalliance/caliptra-rtl
 	branch = main
+[submodule "caliptra-dpe"]
+	path = caliptra-dpe
```

Comment:
I think let's put this in the `runtime/` directory

Reply:
Either is fine with me. Can we also call the directory dpe instead of caliptra-dpe?